components/grid/grouping/aggregates.md

```diff
@@ -82,15 +82,19 @@ To enable aggregates:
                     // you can use aggregates for other fields/columns by extracting the desired one by its
                     // field name and aggregate function from the AggregateResults collection
                     // The type of its Value is determined by the type of its field - decimal for the Salary field here
-                    decimal salaries = (decimal)context.AggregateResults
-                    .FirstOrDefault(r => r.AggregateMethodName == nameof(GridAggregateType.Sum) && r.Member == nameof(Employee.Salary))?.Value;
+                    if (context.AggregateResults.Any())
+                    {
+                        decimal salaries = (decimal)context.AggregateResults
+                        .FirstOrDefault(r => r.AggregateMethodName == nameof(GridAggregateType.Sum) && r.Member == nameof(Employee.Salary))?.Value;
```

Comment:
Are we sure about the code formatting here? Does VS indent this line or not when you auto-format the .razor file? If not, then OK.